openff/interchange/_tests/unit_tests/interop/openmm/_import/test_import.py

```diff
@@ -308,6 +308,24 @@ def test_with_openff_topology(self, sage, basic_top):
             },
         )
 
+    def test_openmm_roundtrip_missing_positions(self):
+        topology = MoleculeWithConformer.from_smiles("CCCCCCO").to_topology()
+        topology.box_vectors = Quantity([4, 4, 4], "nanometer")
+
+        interchange = ForceField("openff-2.2.1.offxml").create_interchange(topology)
+
+        roundtripped = Interchange.from_openmm(
+            system=interchange.to_openmm_system(),
+            topology=interchange.topology,
+        )
+
+        interchange.positions = topology.get_positions()
```

Comment:
(blocking) Should this be an assert or am I missing something?

Reply:
Yep - I had something else locally but in any case this masked an actual issue, which I believe I fixed now: https://github.com/openforcefield/openff-interchange/pull/1391/changes/749d96dd54a631217a28a58b0e347d0a698de9b2#diff-d8ae9e53d3aa3416dbe348b38a0daea9b879d816bc6b3b6175eb2ca0331c3e11R71-R73